core/src/test/java/org/jboss/pnc/build/finder/core/it/FileErrorIT.java

```diff
@@ -114,17 +115,17 @@ void testChecksumsAndFindBuilds(@TempDir File folder) throws ExecutionException,
                     analyzer.getChecksums(ChecksumType.md5),
                     hasEntry(
                             is("ac2a6ab1fbf6afba37789e2e88a916a6"),
-                            contains("jboss-jaxb-intros-1.0.2.GA-sources.jar")));
+                            contains(hasProperty("filename", is("jboss-jaxb-intros-1.0.2.GA-sources.jar")))));
```

Comment:
In this and any other `Test` file where it verifies the property `filename`, it should also verify `fileSize` now.

Reply:
Yes, sorry for my laziness :D